app/javascript/packs/followButtons.js

```diff
@@ -27,12 +32,24 @@ function addButtonFollowText(button, style) {
  * @param {HTMLElement} button The Follow button to update.
  * @param {string} followType The followableType of the button.
  * @param {string} followName The name of the followable to be followed.
+ * @param {string} style The style of the button from its "info" data attribute
  */
-function addAriaLabelToButton(button, followType, followName) {
-  button.setAttribute(
-    'aria-label',
-    `Follow ${followType.toLowerCase()}: ${followName}`,
-  );
+function addAriaLabelToButton(button, followType, followName, style = '') {
```

Comment:
refactor (blocking) - it's definitely great having this as a helper function - but I can see we're actually calling this in a few places with the arguments the wrong way around (e.g. on line 24 we pass in followName and followType in the wrong order).

I think it would be easier to read if we pass these options in an object e.g.

```js
function addAriaLabelToButton({ button, followType, followName, style = '' }) {
```

and then when we call it we can be really explicit:

```js
addAriaLabelToButton({
  button, 
  followName: name, 
  followType: className, 
  style: 'follow'
});
```